tests/worker/workflow_sandbox/test_importer.py

```diff
@@ -19,7 +19,7 @@
 def test_workflow_sandbox_importer_invalid_module():
     with pytest.raises(RestrictedWorkflowAccessError) as err:
         with Importer(restrictions, RestrictionContext()).applied():
-            import tests.worker.workflow_sandbox.testmodules.invalid_module
+            pass
```

Comment:
Was this meant to be removed?

Reply:
Good catch, no this was overzealous `ruff check --fix` autofixing.